go-manual/modules/ROOT/pages/query-simple.adoc

```diff
@@ -0,0 +1,364 @@
+= Query the database
+
+Once you have xref:connect.adoc[connected to the database], you can run queries using <<Cypher>> and the function link:https://pkg.go.dev/github.com/neo4j/neo4j-go-driver/v5/neo4j#ExecuteQuery[`ExecuteQuery()`].
+
+[TIP]
+`ExecuteQuery()` was introduced with the version 5.8 of the driver. +
+For queries with earlier versions, see xref:transactions.adoc[Run your own transactions].
+
+
+== Write to the database
+
+To create a node representing a person named `Alice`, use the Cypher clause link:{neo4j-docs-base-uri}/cypher-manual/current/clauses/merge/[`MERGE`]:
+
+.Create a node representing a person named `Alice`
+[source, python]
+----
+result, err := neo4j.ExecuteQuery(ctx, driver,
+    "CREATE (p:Person {name: $name}) RETURN p",  // <1>
+    map[string]any{  // <2>
+        "name": "Alice",
+    }, neo4j.EagerResultTransformer,
+    neo4j.ExecuteQueryWithDatabase("neo4j"))  // <3>
+if err != nil {
+    panic(err)
```

Comment:
> or maybe Go readers will know that anyway

:arrow_double_up: 

If they don't, they're probably only doing a hobby project. Proper error handling is application-specific (and a hassle) and I don't think we should worry too much about ensuring users don't go the wrong way when it comes to Go practices (rather than driver practices).